[2025-06-13T22:45:26.633+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_projeto.extract_data manual__2025-06-13T22:43:46.978027+00:00 [queued]>
[2025-06-13T22:45:26.642+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_projeto.extract_data manual__2025-06-13T22:43:46.978027+00:00 [queued]>
[2025-06-13T22:45:26.642+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-06-13T22:45:26.657+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_data> on 2025-06-13 22:43:46.978027+00:00
[2025-06-13T22:45:26.661+0000] {standard_task_runner.py:60} INFO - Started process 37 to run task
[2025-06-13T22:45:26.665+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_projeto', 'extract_data', 'manual__2025-06-13T22:43:46.978027+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmphu1dpxm2']
[2025-06-13T22:45:26.668+0000] {standard_task_runner.py:88} INFO - Job 28: Subtask extract_data
[2025-06-13T22:45:26.730+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_projeto.extract_data manual__2025-06-13T22:43:46.978027+00:00 [running]> on host a984848cc8d0
[2025-06-13T22:45:27.031+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kayo' AIRFLOW_CTX_DAG_ID='etl_projeto' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T22:43:46.978027+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T22:43:46.978027+00:00'
[2025-06-13T22:45:29.209+0000] {logging_mixin.py:188} INFO - Arquivo salvo em /opt/airflow/data/dados_api.csv
[2025-06-13T22:45:29.212+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-13T22:45:29.226+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_projeto, task_id=extract_data, execution_date=20250613T224346, start_date=20250613T224526, end_date=20250613T224529
[2025-06-13T22:45:29.256+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-13T22:45:29.303+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
